po/extra/meson.build

```
@@ -0,0 +1,4 @@
+i18n.gettext('extra',
+    args: '--directory=' + meson.source_root(),
+    preset: 'glib'
```

```suggestion
    preset: 'glib',
    install: false
```

This is the keypoint